### PR TITLE
Clarify floating tag use in Mandrel builder images

### PR DESCRIPTION
**Pull request by galderz**
Follows up https://github.com/quarkusio/quarkus/pull/11551.

### CODE REVIEW

File: docs/src/main/asciidoc/building-native-image.adoc [hunk 1]
```diff
@@ -395,11 +395,12 @@ Building with Mandrel requires a custom builder image parameter to be passed add
 ./mvnw package -Pnative -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-mandrel:{mandrel-flavor}
 ----
 
-Please note that the above command points to a floating tag for ease of use,
-and to make sure you get the newest Mandrel version supported by Quarkus.
-Consider using specific tags when going into production.
-Check https://quay.io/repository/quarkus/ubi-quarkus-mandrel?tab=tags[here] to find out about available tags.
-
+Please note that the above command points to a floating tag.
+It is highly recommended that use the floating tag,
```
Comment (gsmet) [inline docs/src/main/asciidoc/building-native-image.adoc:399]:
```suggestion
It is highly recommended that you use the floating tag,
```
?

**Review by gsmet — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/building-native-image.adoc (hunk 1 above)
Comment (jerboaa) [reply docs/src/main/asciidoc/building-native-image.adoc:399]: +1

**Review by jerboaa — APPROVED** [review]
Looks fine (other than the typo @gsmet pointed out).

File: docs/src/main/asciidoc/building-native-image.adoc (hunk 1 above)
Comment (gastaldi) [inline docs/src/main/asciidoc/building-native-image.adoc:399]: Or: 

```suggestion
It is highly recommended to use the floating tag,
```

Comment (gsmet) [reply docs/src/main/asciidoc/building-native-image.adoc:399]: Yeah it's probably better than my suggestion.

**Comment by gsmet** [comment]
Fixed the typo and force pushed, need to reduce the number of open PRs.

**Review by gsmet — APPROVED** [review]

**Comment by galderz** [comment]
Ok, thx @gsmet

**Pull request merged**